CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## CHANGELOG
 
+### 2.0.x
+
+**SDK**
+- *Feature:* Added public interface to create a mini app using mini app id `MiniApp#create(appId:completionHandler:messageInterface)`
+- <span style="color:brown">*Deprecated*:</span> `MiniApp#create(appInfo:completionHandler:messageInterface)`. Your App should instead use `MiniApp#create(appId:completionHandler:messageInterface)`.
```

Comment:
Adding a color the prefixes is a good idea! But we haven't defined what the colors should be yet, so let's remove it for now until we have it standardized. 

@khairul-alam-licon Any thoughts on adding color coding?

Reply:
Done